components/TipOfTheDay.js

```diff
@@ -1,47 +1,76 @@
 import styled from "styled-components";
-
+import Image from "next/image";
+import Herbie from "@/assets/Herbie.png";
 import Link from "next/link";
 
 const StyledCard = styled.div`
   display: flex;
-  flex-direction: column;
+  justify-content: center;
   align-items: center;
+  background: #fcfbf4;
```

Comment:
If the idea is to only set a background color, i would recommend using the `background-color` property. Because the shorthand version used here sets all other, not described properties to their default and there are a lot of default values defined in the shorthand.